src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -4,6 +4,15 @@ public abstract class Machine implements Workable {
     private String name;
     private String color;
 
+    public Machine() {
+
```

Comment:
Please, delete empty line in default Constructor
```suggestion
public Machine() {} 

or public Machine() {
}
```